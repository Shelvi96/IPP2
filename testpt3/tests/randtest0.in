NEW ID

757:0912>27;:0644
83>2227775666
3147:999;8>28;65:78
9224199>385:9
5036>232832
002803;2>008074:66;
:2:;95>904
1821954>49319;9;97
45965:;;>43853
604;490;21>31;;488
21>2
0:;83>;1
44;54;;612>513128
3098>77522
4>009950836
2;45>65;2:
26813348;2>;3:84:;:9
16>508466584
289>800
;3:>;9;9:1243
5;43910272>80:8060
9076;844>9826;69101
;79877>;
0;20:0>:5816
40821>274225
202593407>4;598
7>:
4115781126>4::2816001
066>0183118
6>68986055
7>3554
9847899>0290887
8>22977;2
385;67;;2>833051051;
94:;59>40;;09:6
9>2856093599
9>3566:45
8928:>6
648616:>:30;1178
6782032>92;58934::
4>9641597
65312559>769:
90>2:50651196
7>757::173
628>53
5:>77
15>6
2>144:33
4246966>391
003;794>3;:8465398
52>3;
14929>2:5
9>:169;
29;56>832865
;131;18>72610;
:260;9>9190
5:>;923;1872
9091>8
663684>9284;6438:
;>66127638
304>3268:
2635478466>959378
:5715;3021>0
6:306;0;9>2644130:59
:6>67;77;8213
7>:
3>9251
8525>;494:439
69:>183761
4:51>02967463
:5:636>4409075
8>2
5469>7
5;:95023:>93657392
320:59607>7;9
9:41570283>5:42485
974618557>765
:7:9>592524140:
2;9>53630
7>85
;43838:5>0
7515;7>826
09428331>254
8>05:56
2;31701>6557
8343>5529;;14
497:3:5>9
52>:823
:8735>92707
9;04136;;4>568
22>:0;57415
959>6952209
0129;:>2
7056:>2
1487>153685657
:939;0337>06650
:565;7325;>;8
::7>769329;235
3::0>737:5
154>327;
7912;:>61658871
740521;;90>305273312
2868>7818180
66;5589>;95
8819;2>;1
588;41;879>:6064891
3:2475:>53:2;:5705
;2;:9>810:26
48906;>;3
52>;:4467:
978:>887
1>3:;992
0853085>4;33852
37532>4122533;78
53>18
6;;17833>:864
:>2436;21
:6:90:1694>80203589
8;7;7:5:>:6
9809>5749
59854>5576425352
:>011
5:7>940392
6>934498;
448>612;224
:726>676
14;88>69705
939:>84
4910074622>76387;6
:61>74:9
624;>63;1
98>57
1760123074>37
45;9827;>02387987:
529099>28;1
76883;3694>4481
3389:>1;6852;6
7>47
7;24739>4247
48019;7:5>93984644
7;:26;5:51>3967:8;9:
;>;888
408>;656524:
2>11549
1>04:0
:072>35012:652
99>2:06;90:
48197591>07:298062
3;8:>14
:>2
5;56::33>4
6:42:>54680
:50:7>176440
6893449>84469
05143:566>;54
981742>8
0>24;
1;>:46633011;
5631;::11>815:58;
;184;6;>6439:07
1673>828
7663682;>::310:
7;37;00584>4:
65>40:129:57
:;3463;560>;8194;2:82
80839:083:>72126:
68;690:13>198::85;
5>977
:1:8;:2>28926257
37153:8178>38
59210438>3
91>692188
067>6349
82>2316:
11>22183
:03842>50;292
3746028>39
5386;7>;3439009;
650>44
4739>480
8::4617:84>9
17;;80>47032166::
:88229>4697
@861;217;0;220002:61:6703991:401476410;00:772775:5:0;9576289
@9::88298521798:877:19;464;5682094250569282:9206978287723679611754461;7771553934
@;7476;2460305:5133622217;
@:917952240164;6;;4:99544;;:06935:4489621:386666:24701380264
@776:4::580;:77
@55:30
@20768;8649137;7911914703068:105404:04::00037335586:0233213424968681:7;:7356;0388::8
@13667:9387;2444346:;38:40602512:885366:6120568022:2:;023;2587;647;;59:4;08869
@93;31:3316::259998012:;:1;:9976:;:1009
@23606;271:;2411224145024;0541687:2;851760:0405;694:741;418063015615;64:62:3282996;4:036606076
@40;9455207432053:32213:0126195;19:214738603808;236549382577607;955;1469:607;0:134605187:778
@2;9740186:;5;:4;897587297203982;0;:43;52:471234203;822::4:51:;423276648807922583983;:116
@:76:;815773;8;;705::21724:4:0:5:5;88015719;9024582327:848060:;;33;33102611324;746:65
@2982714707232582;2419:919177;07527798;80::8478:7;60049112
@011;67;58;854:7067846;;:077111212350:49:353;73;1209038:385996;206748316;:
@:55::;27;589146:14938330;61553;4086:789:557
@154295:50193;29550199774793062;883:9103
@145176:46;23:105:395981543354:692321946838;19;:868544:181459647171296851447685661;
@:94:204078;87424:394;6;564;0:238;7:672:62162596808036;9438853;16:00:6:41;3390154197;88339490726
@6;8590613127673083343;;0304:3;81:46;9444564457812;9::9:
@9304606850;254622::8:6146;2805:10266246;45111884:608555347;4016034::84942;63;6
@14;29570164:758;102;06840;24283817:209753;324;65
@883690782;336;81125:9;186856221:39409;0327:96627189:7;;178;921;5:35;3:295564393491344693214463:8
@771
@1;;6932:667;9;331696:201480;719048;231017819;44432:18:70:7;9082581736;557:2;6;391231421794815:
@13591967441720;455;611;824939;223;38893555009343831:41:69716730274:3520
@;138
@7352536:147955819:8583;28611929452:2940;0;0940:6::;:6212867984567384;17;0
@99;;7126001561;1;05464861770;00813;8512557:;91005:4324:70983103232:8851103119112
@6:62816919;105534139
@412533;462248542;539859224:377208993008:32237:9;8584:9:468;1
@25437737722:8:9;;874836:2:4::564
@27:1:9944;128419;:5643837;15558;4:0271;;04518;:;9755;28:5172:712519538870688567664;5:073
@7502:60439;;96337;05:6;7:85:493;675925196014:47580;:::587;
@3026:937;109742;36;02;3512::1108126409;3:4468693584;;74513768722:862:998126:04154958591:4855377
@8176:9
@4;16;5;9992563:40615214795777413729:;9;96164990132:93294;4;
@04:;7;16817666:3338:5688050;136578867958:465:9050985745819;6053;5;658;5;7;4:88309;9
@7250;463293;0189418;5495056:837864890752886912:54:
@92626308;47;2383478:344057223
@;469:50;1371394300:1;15763689998539481725
@707337410766201992:63410022;8;57;0:2;67011:423135;18721783:823353273867:75698;01252:839
@6850;857242;:9192;;2:3709925005:0:2;7;:130059
@3342529176:933995978167;834081:3929297741:
@51326416902105505::175:2299:;738062;33507;578:829:7835:9707:;;64983030530::8962:9960792299
@88:06:7:583806703575243730959:9:;7295:7;:372922409926419527614800;99180;38502
@82999322475;5:82:01;96;:53:274947:5857:26710516717:31:5:5304
@10646695530054:::5094726043787618152;7;4:;9777694:6890694980522577;76:352449;:;80589:3:24631
@9608973830:74;;7:7;8796:783638743;10;8028:142430338;469334
@:183888074114320974::;:935;:0702089885179201976762981;688960863970576:77471267
@09:9:17::48:03192671498;28082;938;4:0;8:3917469608;4588;8072049871;00;:783
@0:4::4:71:31638835:40;30063736238:6764682910
@9006:8;:3329:9054816201:61472443442701974;4309089533:81815471829444851324;958115:
@0
@4155:14;1715;519;83370085253:607;94229534603450313:07;80128308:;931;476816;9;400;34;21373
@23;16888;0750033:2726;5;:74567592420520467260912094:0:129578409:5;;25;:;
@49;2:16;9437659;19856625542535876798
@3890;9:9:1279714::9606192931:;16;
@:;4802921796;;40687;5980
@;;49;;72372948:3546907702693796:1370266:187946512;2769708583621790;0:5:35941
@93:9
@57748314:0:3783701428:;0;7:69;466;:264:4417312:1773356341532478;6750388793::91
@48:;5421:0404;939811;99:
@40815;9598:13433847053954;4186;567606853488;002882815::99629012:81:
@98958958978999267078695628921;8:7973703800:19088174;59:;976:62064813053051262227166:38:53432:882499
@219;23589;3:5291:7663181188555206;316193051:;:;199702
@643211;31:64;0503:598477:985925768909972727:64:224;:
@:927989414212233:52400:9203:;63:7
@591616;305638482816:28;572;:0;3:89
@977070:286;2648565430:7:;6512293257956460387;549:002:741592744269:764;086175:;20;72:2:;07
@;3:29;09588:43;4093717239850806888:97265:336:6::76981;3:88:8094856:
@84::;551374:11:61181179182258029847894100525700058167:874215272:09:1202298;49386491;01;83419;37;1:
@3279;3084443864;:5:602896;4546:;8900518954012840537597236;1393
@613:6467120658:9254;336::535277082:2689;3958:454123459776:01901524;141072630214
@700417089497424;:40350:7
@600;88608:272287333902886;1;10974:6564563719198450296:551:422127300
@5970509:263;:58875103529;842988263238312981;5184:9417;:677
@878:5440086692;9009426935853744400698
@:0;460:51:9:3;90
@2046;823062
@310344242765948850784;63;8;02525621
@:3:4289;590:1765:01904139787191;4;8785657:38526767973::0::;;0034;;3
@8:4388462:58923055034;64;90284256:06
@4406:934439:742:;:102496:648:;777;1585:18;;485673879088:60:
@394;99892::3:1;6655114315;33;60778:466685437:2588221:5
@342;7836
@0406;66832754414:76845407388:
@4:;48237::993518:;48;81390:98930;6588076:475981;;9;;646755816;65:316385
@40:61390193;526:72455::314:84719737064571867:099652435;89141892409472927:1
@05820188517529245;:148:;:75996424044104:6;888:42133
@314512220:44280814674674406672;27870293207;28;:189843788;36:990457979029;1
@;020
@3046016;84:52:17::7;2986:3929:302;720568:45067;896747725;;08179468:75
@73017;077153512;61;:205380;7;436047777229896146759:;932:7196:412009704294;7575412204:
@25830:7508523565629674;39262:894197506:4246:147868255596;7893918;023;071823::3:5;4281;31:325
@824847786198877;94356;6:2706:6;607686640;5
@700718:629144852:358303;:17:013753673856:6227;89216551832856115:809083::104800:214
@6901;0;510:4021034:1:;6:3685009:12402798;00;6
@09::3011:4;723;3215:5857447835137;6:;732429::01017:;336;75;2:065447732979:57;7;46931490363
@5332;727;8865740:3;2906:872;255;00508;;787516164497125;:45:::79;7737626294
@456023775478946735964
@902;:902980392:898184212187:23633926:68738:420408504;
@6195339:64;0:97782
@6:::2270
@;;86493810:;590
@:;6652434256
@90118;5665;73:44:;61646009437288299;07970868624913
