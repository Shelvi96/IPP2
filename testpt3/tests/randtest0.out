0
0
7400020102000
0
50
0
0
0
4611260298682317406
0
0
0
0
0
0
2184883536018800640
7619528736248954880
0
0
0
0
6498694262295625728
0
0
0
0
0
0
0
163165005
0
9323033715161793536
0
0
0
0
11723642724560553534
1052967208012087459
13474770085092524032
426497035737441559
8854237670985957376
0
1028579933644914688
14017131683283730432
0
0
0
0
5558049960934376678
0
0
7771858358516056064
0
0
0
0
2702482110017956477
3787514887180787250
2388897082737
0
0
0
730013120100
0
0
12075212381911333722
3458764513820540928
12159981012310294528
16792563631756673024
0
0
0
0
0
2386268084882
0
0
17088873063783971511
4617
0
9144921348324737012
0
9628791040438468608
0
0
0
388905518572107768
0
0
14123288431433875456
0
0
0
0
0
14017131683283730432
0
1028579933644914688
0
0
354477709
4611686018427387904
2345092
0
741
0
13474770085092524032
